install_mac_requirements.sh

```diff
@@ -129,43 +138,43 @@ if ! brew help >/dev/null 2>&1 || [[ "$PATH" != *"$(brew --prefix)/bin"* ]]; the
     
     if $echoOn; then
         printf "> if [ -d \"/opt/homebrew\" ]; then\n"
-        printf ">     brew_prefix=\"/opt/homebrew\"\n"
+        printf ">     brewPrefix=\"/opt/homebrew\"\n"
         printf "> else\n"
-        printf ">    brew_prefix=\"/usr/local\"\n"
+        printf ">    brewPrefix=\"/usr/local\"\n"
         printf "> fi\n\n"
     fi
     
     # Retrieve brew prefix
     if [ -d "/opt/homebrew" ]; then
-        brew_prefix="/opt/homebrew"
+        brewPrefix="/opt/homebrew"
     else
-        brew_prefix="/usr/local"
+        brewPrefix="/usr/local"
     fi
     
     if $echoOn; then
-        printf "> printf \"\\\neval \\\\\"\\\$(\\\\\"%%s/bin/brew\\\\\" shellenv)\\\\\"\\\n\" \"\$brew_prefix\" >> ~/.bash_profile\n\n"
+        printf "> printf \"\\\neval \\\\\"\\\$(\\\\\"%%s/bin/brew\\\\\" shellenv)\\\\\"\\\n\" \"\$brewPrefix\" >> ~/.bash_profile\n\n"
```

Comment:
If this is the same string that would go into both .bashrc / .bash_profile AND .zshrc then pull it into a single string that is assigned to a variable and re-use it.